packages/patternfly-4/content/design-guidelines/usage-and-behavior/filters/design.md

```diff
@@ -0,0 +1,220 @@
+When filters are applied, filter chips may be used to provide feedback.
+
+**Basic filter chips**
+As filters are applied, the value will be displayed as a filter chip. Each filter chip has an ‘x’ to remove the filter, or users may click “Clear filters” to clear all applied filters at once.
+
+![image11](img/basic-filter-chips.png)
+
+**Complex filter chips**
+Filter chips may be used to show Boolean relationships between different attribute-value pairs. Individual filter chips can be removed by clicking the ‘x’ in each chip, and entire filter groups can be removed using the group ‘x’.
+
+![image12](img/attribute-value-multi-chips.png)
+
+![image13](img/faceted-combo-chips.png)
+
+* Filters between attribute categories should be combined with a boolean AND operator.
+* Filters within a category are shown grouped together and should be combined with a boolean OR operator.
+  * **Example**: An item must be Stopped OR Down AND have Samsung OR Hewlett-Packard as a vendor.
+* If the list is not otherwise sorted, items that satisfy a higher number of filters should be shown higher up the page.
+  * **Example**: Items that have both Samsung and Hewlett-Packard as vendors are listed before items that only have Samsung or Hewlett-Packard.
+
+In the example above, the result will be to show all items that have a status of stopped OR down AND are from vendor Samsung OR Hewlett-Packard AND match the keyword.
+
+#### When to use
+Use filter chips when:
+* The value of the applied filter is not shown on the control itself.
+* There is more than one value for a given facet.
+* There are boolean filter rules being applied.
+
+#### When not to use
+Do not use filter chips when:
+* The current settings of the controls are sufficient to convey applied filters.
+
+
+
+## Variations
+### Multiple filters
+You can use multiple filters in combination to create a custom filtering experience.
+
+![image14](img/multiple-filters.png)
+
+#### When to use
+Use multiple dropdown filters when you only have a handful of attributes that you want to show. Only use this filter when space allows; otherwise consider using a faceted filter.
+
+#### Behavior
+Filter queries are applied when the user selects or deselects an item in the list or selects a date/time range. A clear filters action to the right of the filters will remove all applied filters. If there are no applied filters, the icon will be disabled.
+
+
+### Complex faceted filter
+You can create a complex faceted filter by incorporating elements such as a search field into the dropdown component.
+
+![image15](img/faceted-complex.png)
+
+#### When to use
+Use a complex faceted filter when you have limited space and multiple attributes with a variety of input types, for example, and search field and checkbox selects.
+
+#### Behavior
+Filter queries are applied when the user selects or deselects an item in the list, or types a query in the input field and presses Enter.
+
+
+
+## Related components and demos
```

Comment:
We should include Toolbar in the Core list. Looking at this raises a couple or questions to me: 

* Is it helpful here to list every component that can be used? The list here is mostly correct, but for example does a user looking at this page want the exhaustive list of all sub-components or would I just want to have a link to say the Filter Demo where I can see live examples and then from there see what's required to implement?
* This is the first instance where the documentation is actually out ahead of implementation. On the React side especially, there are definitely more components used than are listed here. Many of the required components don't exist yet. There may be many cases like this in the future. Should we be listing components that are still WIP? If not, it gives the impression that this is a complete list when it's not. How to we make it painless to keep this list updated when components are released? Another option is to not publish these guidelines in the nav until development work is done.

@rachael-phillips @stacymcauliffe interested in your thoughts.

Reply:
Hi @mcarrano I agree, I don't think we should list any components that haven't been released. Also, perhaps we'll leave the design guidelines off the site until the component is ready which may help clarify what to list. I have less insight on the first bullet. I think linking to the Filter Demo is the most simple way to provide pointed guidance.